ponyo/helper_vae.py

```diff
@@ -32,7 +31,7 @@
 # The below is necessary for starting core Python generated random numbers
 # in a well-defined state.
 
-rn.seed(12345)
+np.random.seed(12345)
```

Comment:
I saw one of your earlier comments that said that random wasn't a package available in pypi. Shouldn't random be built into the python language? Is `random2` a package designed to overwrite the normal behavior of the random package?

Reply:
I believe you're right about random being pre-packaged with python. I wasn't aware of that at the time when I was making the list of required packages so I ran into an issue when `random` wasn't found in pypi.

Reply:
Are you sure that np.random.seed and random.seed do the same thing? In the very distant past (like, 10+ years ago), I think that the random number generators were not shared between python and numpy. You would have to set both (or just the one you were using) to get the desired behavior. Are you observing that the RNG behaves as you wish with either set, and that not setting one or the other produces undesired behavior?

Reply:
You are correct. This is a case of human error as I was refactoring imports. There should be two distinct random seeds set using random and numpy libraries. 

As discussed, I will also try to add a pytest to test reproducible outputs and see how that goes